<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computer Vision </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color:rgb(0, 0, 0);
}

QScrollBar:vertical {
	width: 4px;
	margin: 0;
}

QScrollBar::handle:vertical {
	min-height: 10px;
	background-color:rgb(27, 27, 27);
}

QScrollBar::add-line:vertical {
	background: none;
	height: 45px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}

#scrollAreaWidgetContents QPushButton{
	color:white;
	font-size:14px;
	text-align:left;
	padding-left:18px;
	height:56px;
}

#scrollAreaWidgetContents QLabel{
	color:rgba(0, 208, 255,0.7);
	font-size:18px;
	padding:20px;
	padding-left:13px;
	border-bottom:1px solid rgb(206, 206, 206);
}

#SettingsSideBar QPushButton{
background-color: rgb(0, 0, 0,0);
color:white;
text-align: left;
padding-left:25px;
}
</string>
   </property>
   <widget class="QFrame" name="SettingsSideBar">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>9</y>
      <width>201</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/**background-color:qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.517091, y2:1, stop:0 rgba(0, 0, 0, 215), stop:1 rgba(64, 64, 64, 213));**/
border:none;
border-radius:5px;
background-color:rgb(12, 12, 12);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="OpenImageBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 0,0);
height:50px;
margin-bottom:8px;
}

QPushButton:hover{
background-color:rgb(0, 130, 220);
}</string>
     </property>
     <property name="text">
      <string>Open Image</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveImageBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 0,0);
height:50px;
margin-bottom:8px;
}

QPushButton:hover{
background-color:rgb(0, 130, 220);
}</string>
     </property>
     <property name="text">
      <string>Save Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AdjustImageBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 0,0);
height:50px;
margin-bottom:8px;
}

QPushButton:hover{
background-color:rgb(0, 130, 220);
}</string>
     </property>
     <property name="text">
      <string>Adjust</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UndoBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 0,0);
height:50px;
margin-bottom:8px;
}

QPushButton:hover{
background-color:rgb(0, 130, 220);
}</string>
     </property>
     <property name="text">
      <string>Undo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ResetBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 0,0);
height:50px;
margin-bottom:8px;
}

QPushButton:hover{
background-color:rgb(0, 130, 220);
}</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MenuToggleBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:none;
padding:0;
text-align:center</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/Programmation/DEV WEB/web dev mine/learn html css js only/images/icons desktop/icons8-menu-240.png</normaloff>D:/Programmation/DEV WEB/web dev mine/learn html css js only/images/icons desktop/icons8-menu-240.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="SwitchViewBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 0, 0,0);
height:50px;
margin-bottom:8px;
}

QPushButton:hover{
background-color:rgb(0, 130, 220);
}</string>
     </property>
     <property name="text">
      <string>Switch View</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>20</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeHorCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>D:/designs-web/31385783667_ac3590d0cf_b.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="FiltersBtn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>590</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:8px;
color:white;
font-size:16px;
background-color: rgb(0, 30, 50);
}

QPushButton:hover{
background-color:rgb(0, 130, 220);
}
</string>
    </property>
    <property name="text">
     <string>No Filters</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>231</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0,0.4);
border:none;
border-radius:12px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>227</width>
       <height>2128</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="NoEffectsBtn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>No Effects</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLight">
        <property name="styleSheet">
         <string notr="true">background-color:rgba(1, 230, 255, 40);</string>
        </property>
        <property name="text">
         <string>Light</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BrightnessBtn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>Brightness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ContrastBtn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>Contrast</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HighlightsBtn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>Highlights</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ShadowsBtn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>Shadows</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelColour">
        <property name="styleSheet">
         <string notr="true">background-color:rgba(1, 230, 255, 40);</string>
        </property>
        <property name="text">
         <string>Colour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="InvertBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>Invert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaturationBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>Saturation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="WarmthBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>Warmth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TintBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>Tint</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="GrayScaleBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(1, 234, 255);
background-color:rgba(0, 179, 255, 50)
}</string>
        </property>
        <property name="text">
         <string>GrayScale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelColour_2">
        <property name="styleSheet">
         <string notr="true">background-color:rgba(1, 230, 255, 40);</string>
        </property>
        <property name="text">
         <string>Histogram</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HistBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(0, 255, 183);
background-color:rgba(0, 255, 217, 40)
}</string>
        </property>
        <property name="text">
         <string>Histogram</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HistEqualizerBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(0, 255, 183);
background-color:rgba(0, 255, 217, 40)
}</string>
        </property>
        <property name="text">
         <string>Equalizer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HistStretchBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(0, 255, 183);
background-color:rgba(0, 255, 217, 40)
}</string>
        </property>
        <property name="text">
         <string>Stretch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelColour_3">
        <property name="styleSheet">
         <string notr="true">background-color:rgba(1, 230, 255, 40);</string>
        </property>
        <property name="text">
         <string>Binarization</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ManualTresholdingBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(0, 255, 183);
background-color:rgba(0, 255, 217, 40)
}</string>
        </property>
        <property name="text">
         <string>Manual Binarize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OtsuTresholdingBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(0, 255, 183);
background-color:rgba(0, 255, 217, 40)
}</string>
        </property>
        <property name="text">
         <string>Otsu Binarize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelColour_4">
        <property name="styleSheet">
         <string notr="true">background-color:rgba(1, 230, 255, 40);</string>
        </property>
        <property name="text">
         <string>Blur Filters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="GaussienBlurBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(255, 0, 242);
background-color:rgba(183, 0, 255, 40)
}</string>
        </property>
        <property name="text">
         <string>Gaussien Blur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MeanBlurBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(255, 0, 242);
background-color:rgba(183, 0, 255, 40)
}</string>
        </property>
        <property name="text">
         <string>Mean Blur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MedianBlurBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(255, 0, 242);
background-color:rgba(183, 0, 255, 40)
}</string>
        </property>
        <property name="text">
         <string>Median Blur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelColour_5">
        <property name="styleSheet">
         <string notr="true">background-color:rgba(1, 230, 255, 40);</string>
        </property>
        <property name="text">
         <string>Morphology</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ErosionBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(255, 200, 0);
background-color:rgba(255, 162, 0, 40)
}</string>
        </property>
        <property name="text">
         <string>Erosion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DilationBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(255, 200, 0);
background-color:rgba(255, 162, 0, 40)
}</string>
        </property>
        <property name="text">
         <string>Dilation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OpeningBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(255, 200, 0);
background-color:rgba(255, 162, 0, 40)
}</string>
        </property>
        <property name="text">
         <string>Opening</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ClosingBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(255, 200, 0);
background-color:rgba(255, 162, 0, 40)
}</string>
        </property>
        <property name="text">
         <string>Closing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelColour_6">
        <property name="styleSheet">
         <string notr="true">background-color:rgba(1, 230, 255, 40);</string>
        </property>
        <property name="text">
         <string>Edge Extraction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="GradiantBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(76, 255, 0);
background-color:rgba(49, 255, 3, 40)
}</string>
        </property>
        <property name="text">
         <string>Gradiant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SobelBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(76, 255, 0);
background-color:rgba(49, 255, 3, 40)
}</string>
        </property>
        <property name="text">
         <string>Sobel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="KirschBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(76, 255, 0);
background-color:rgba(49, 255, 3, 40)
}</string>
        </property>
        <property name="text">
         <string>Kirsch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RobinsonBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(76, 255, 0);
background-color:rgba(49, 255, 3, 40)
}</string>
        </property>
        <property name="text">
         <string>Robinson</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LaplacianBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(76, 255, 0);
background-color:rgba(49, 255, 3, 40)
}</string>
        </property>
        <property name="text">
         <string>Laplacian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SiftBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(76, 255, 0);
background-color:rgba(49, 255, 3, 40)
}</string>
        </property>
        <property name="text">
         <string>Sift</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HarrisBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(76, 255, 0);
background-color:rgba(49, 255, 3, 40)
}</string>
        </property>
        <property name="text">
         <string>Harris</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HoughBtn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
color:rgb(76, 255, 0);
background-color:rgba(49, 255, 3, 40)
}</string>
        </property>
        <property name="text">
         <string>hough</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="AdjustArea">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>509</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="AdjustToolFrameLayout"/>
   </widget>
   <widget class="QLabel" name="Image_copy">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>20</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeHorCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>D:/designs-web/31385783667_ac3590d0cf_b.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>Image_copy</zorder>
   <zorder>SettingsSideBar</zorder>
   <zorder>Image</zorder>
   <zorder>FiltersBtn</zorder>
   <zorder>scrollArea</zorder>
   <zorder>AdjustArea</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
